src/libstd/alloc.rs

```diff
@@ -8,18 +8,83 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! dox
+//! Memory allocation APIs
+//!
+//! In a given program, the standard library has one “global” memory allocator
```

Comment:
Some smart quotes snuck in here.

Reply:
Is that bad?

Reply:
We try to use smart quotes in the docs, but the API reference is all over the place. The book uses them 100%. Either way is fine by me.